src/widgets/lv_label.h

```diff
@@ -69,11 +69,11 @@ typedef struct {
 #endif
 
     lv_point_t offset; /*Text draw position offset*/
-    lv_label_long_mode_t long_mode : 3; /*Determinate what to do with the long texts*/
+    lv_label_long_mode_t long_mode : 3; /*Determine what to do with the long texts*/
     uint8_t static_txt : 1;             /*Flag to indicate the text is static*/
     uint8_t recolor : 1;                /*Enable in-line letter re-coloring*/
-    uint8_t expand : 1;                 /*Ignore real width (used by the library with LV_LABEL_LONG_SROLL)*/
-    uint8_t dot_tmp_alloc : 1; /*1: dot_tmp has been allocated;.0: dot_tmp directly holds up to 4 bytes of characters*/
+    uint8_t expand : 1;                 /*Ignore real width (used by the library with LV_LABEL_LONG_SCROLL)*/
+    uint8_t dot_tmp_alloc : 1;          /*1: dot_tmp has been allocated;.0: dot_tmp directly holds up to 4 bytes of characters*/
```

Comment:
Please don't apply the suggestion from CI; it doesn't make sense. Perhaps we can just shorten the comment?

```suggestion
    uint8_t dot_tmp_alloc : 1;         /*1: dot is allocated, 0: dot directly holds up to 4 chars*/
```